crates/neon-runtime/src/napi/mem.rs

```diff
@@ -1,3 +1,12 @@
-use crate::raw::Local;
+use crate::raw::{Env, Local};
+use crate::napi::bindings as napi;
 
-pub unsafe extern "C" fn same_handle(_h1: Local, _h2: Local) -> bool { unimplemented!() }
+pub unsafe extern "C" fn same_handle(_lhs: Local, _rhs: Local) -> bool {
+    panic!("PartialEq is deprecated with N-API backend");
```

Comment:
Can we move this method? We removed everything else that we couldn't or weren't going to implement.